controlplane/kubeadm/controllers/kubeadm_control_plane_controller.go

```diff
@@ -23,6 +23,7 @@ import (
 	"time"
 
 	"github.com/blang/semver"
+	"github.com/coredns/corefile-migration/migration"
```

Comment:
I think this is probably fine for the short term, but we should definitely create an issue to track decoupling this from Cluster API a bit more. This will require us to release a new version of Cluster API to track newer releases of coredns.

Reply:
💯

Reply:
We can add a verify script later to do this